steal-qunit.js

```diff
@@ -111,8 +111,8 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		
-		if(typeof QUnit.start === "function") {
+
+		if(QUnit.assert && QUnit.assert.ok && QUnit.assert.async) {
```

Comment:
Yes we can do that, we need to just make sure to add the check for the version in which functionality is changed. I am not sure if that is version 2.

By going through the change log I have found this. not sure what does it mean.
![image](https://cloud.githubusercontent.com/assets/3850812/25421749/e0bea306-2a7b-11e7-98e3-3b223fe02ebd.png)

https://github.com/qunitjs/qunit/blob/2.3.2/History.md#211--2017-01-05